resources/overrides/library/sap.ca.ui.d.ts

```diff
@@ -0,0 +1,35 @@
+declare module "sap/ca/ui/ChartColor" {
+	import ChartColor from "sap/ca/ui/library";
```

Comment:
If I'm not mistaken, `ChartColor` is a **named export**, not the default export.

Reply:
Thanks! Now, should be fine